recipe/meta.yaml

```
@@ -27,38 +28,57 @@ outputs:
         - python
         - packaging
         - pandas >=1.4.0
+        - shapely >=2.0.0
+        - numpy >=1.22
+        - pyogrio >=0.7.2
         - pyproj >=3.3.0
-        - shapely >=1.8.0
-        - fiona >=1.8.21
+      run_constrained:
+        # geodatabase access
+        - psycopg2-binary>=2.8.0
+        - psycopg-binary>=3.1.0
+        - SQLAlchemy>=1.3
+        - matplotlib >=3.5.0
+        - matplotlib-base >=3.5.0
```

Does this need `matplotlib` as well as `matplotlib-base`?

Does `{{ name }}` (below) require `matplotlib` and `{{ name }}-base` just `matplotlib-base`?

The arrangement of these seems odd.

---

A user might want to install `geopandas-base` and can choose to install `matplotlib` or `matplotlib-base` (we do use `matplotlib-base` in `geopandas` below for example).

---

OK, but does `matplotlib-base >= X` imply `matplotlib >= X`?  I _think_ it does:

```
$conda search matplotlib=3.9.2=py39*_1 -i
...
dependencies: 
  - matplotlib-base >=3.9.2,<3.9.3.0a0
  - python >=3.9,<3.10.0a0
  - tornado >=5

```

---

I'll remove `matplotlib` then
```suggestion
        - matplotlib-base >=3.5.0
```